packages/wallet-ui/src/components/ui/organism/MinVersionModal/MinVersionModal.view.tsx

```diff
@@ -1,28 +1,28 @@
 import {
-  Description,
   StarknetLogo,
+  MetaMaskLogo,
   Title,
   Wrapper,
 } from './MinVersionModal.style';
-import { useMultiLanguage } from 'services';
+
+import { ConnectButton } from '../ConnectModal/ConnectModal.style';
+import { useMultiLanguage, useStarkNetSnap } from 'services';
 
 export const MinVersionModalView = () => {
   const { translate } = useMultiLanguage();
-
+  const { completeUpgradeSnap } = useStarkNetSnap();
   return (
     <Wrapper>
       <StarknetLogo />
       <>
         <Title>{translate('newVersionAvailable')}</Title>
-        <Description>
-          {translate('installLatestVersion')}
-          <ul>
-            <li>{translate('deleteCurrentVersionMetaMask')}</li>
-            <li>{translate('refreshPage')}</li>
-            <li>{translate('connectToMetaMask')}</li>
-          </ul>
-          {translate('accountRecoveryInfo')}
-        </Description>
+
```

Comment:
We should maybe keep a small description 

```suggestion
<Description>
          {translate('upgradeLatestVersion')}
</Description>
```

And then adding 

en.json
```json
    "upgradeLatestVersion": {
      "message": "To use this dapp, please upgrade to the latest version by clicking the link below:"
    },
```

fr.json
```json
      "upgradeLatestVersion": {
        "message": "Pour utiliser cette dApp, veuillez mettre à jour vers la dernière version en cliquant sur le lien ci-dessous :"
      }
```

wdyt ?

Reply:
good idea, just change a bit , instead of link, but button